vizro-core/src/vizro/models/types.py

```diff
@@ -96,7 +99,14 @@ class JsonSchemaExtraType(TypedDict):
     mode: str
 
 
-def validate_captured_callable(cls, value, info: ValidationInfo):
+class AllowedCapturedCallable(NamedTuple):
```

Comment:
```suggestion
class UndefinedCapturedCallable(NamedTuple):
```

Plus similar renaming throughout. Because the `allowed_undefined_capatures_callables` is really a list of `UndefinedCapturedCallable`s that are allowed rather than `AllowedCapturedCallable`s that are undefined.

I would also suggest we do `allow_undefined_captured_callable` that than `allowed` because it leaves open the possibility of `allow_undefined_captured_callable = True` if we want to do it in future, and it still makes sense as an argument name if it's just a list of allowed things.

Reply:
N/A

Reply:
ah im sorry i had overlooked the second paragraph